lib/taurus/core/tango/tangoattribute.py

```diff
@@ -796,7 +797,9 @@ def push_event(self, event):
                                 rel='4.3.2')
                 self.fireEvent(etype, evalue, listeners=listeners)
             else:
-                manager.enqueueJob(self.fireEvent, None, etype, evalue,
+                manager.enqueueJob(self.fireEvent, None,
```

Comment:
It would be better and more consistent to pass the callback arg as a kwarg (i.e., use `callback=None` instead of `None`)